app/models/spree_multi_vendor/spree/product_property_decorator.rb

```diff
@@ -11,4 +11,4 @@ def property_name=(name)
   end
 end
 
-Spree::ProductProperty.prepend Spree::ProductPropertyDecorator
\ No newline at end of file
+Spree::ProductProperty.prepend SpreeMultiVendor::Spree::ProductPropertyDecorator
```

Comment:
missing empty line

Reply:
oh, I'll add it now. Thank you

Reply:
919cbba0b19b257a02c116989bc38b266fd848fc